### PR TITLE
Add Korean language for Mod Menu Helper

### PR DESCRIPTION
**Pull request by AlphaKR93**

### CODE REVIEW

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+    "modmenu.dropInfo.line1": "FO 모드는 §9✎§r 또는 §8✎§r, 대체제는 §c*§r로 표시됩니다",
+    "modmenu.dropInfo.line2": "다른 모드는 지원되지 않습니다",
+    "modmenu.summaryTranslation.slight-gui-modifications": "애니메이션, 메뉴의 모드팩 버전\n§9✎§8 애니메이션, 색, FPS 인디케이터, ...",
+    "modmenu.summaryTranslation.advancementinfo": "커지고 스마트해진 도전과제 창\n§9✎§8 색, 정보 칸 크기, ...",
+    "modmenu.summaryTranslation.advancements-enlarger": "§c*§r 커진 도전과제 창\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.animatica": "Optifine의 애니메이션 텍스쳐\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.antighost": "/ghost 명령어로 Ghost 블록 제거\n§9✎§8 키바인드",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/ko_kr.json:8]: You might have misunderstood what "ghost blocks" are, so check this video: https://www.youtube.com/watch?v=mHhuU9j4ORo

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "modmenu.summaryTranslation.betterbeds": "§c*§r 빨라진 침대 렌더\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.bettermounthud": "말을 타고있을 때 허기와 경험치 표시\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.bobby": "서버에서 계속 청크가 보이도록 로드\n§9✎§8 블록 엔티티, 청크 어둡게, 캐시 저장, ...",
+    "modmenu.summaryTranslation.borderlessmining": "전체 화면에서 화면 이동 (opt-in)\n§9✎§8 모드 토글, macOS 지원, 커스텀 크기, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/ko_kr.json:12]: "Opt-in" means "can be enabled manually if wanted". Perhaps there is also Korean phrase for that?

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "modmenu.summaryTranslation.citresewn": "Optifine의 커스텀 아이템\n§9✎§8 모드 토글, 깨진 팩 허용, ...",
+    "modmenu.summaryTranslation.colormatic": "Optifine의 커스텀 색\n§9✎§8 맑은 하늘/공허, 부드러운 조명",
+    "modmenu.summaryTranslation.continuity": "Optifine의 연결된 텍스쳐\n§9✎§8 모드 토글, 메소드, 리소스 팩",
+    "modmenu.summaryTranslation.cullleaves": "Optifine의 \"스마트 나무\"\n§9✎§8 모드 토글, 리소스 팩",
+    "modmenu.summaryTranslation.cem": "Optifine의 커스텀 엔티티 (알파)\n§9✎§8 옵티파인 지원, 모델 고침",
+    "modmenu.summaryTranslation.dcch": "여러 월드에 걸쳐 입력된 기록 유지\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.dynamicfps": "백그라운드에서 FPS 감소\n§9✎§8 모드 토글, 호버시 복구, ...",
+    "modmenu.summaryTranslation.enhancedblockentities": "빠른 상자, 침대, 표지판 등 렌더\n§9✎§8 개선할 블록, 메소드",
+    "modmenu.summaryTranslation.entityculling": "벽 뒤 엔티티 렌더 안함\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.entity_texture_features": "Optifine의 랜덤/Emissive 몹\n§9✎§8 랜덤 몹, Emissivity, 눈 깜빡임, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/ko_kr.json:23]: "Emissive" as in "glowing" or "giving off light". Example: https://media.forgecdn.net/attachments/428/176/2022-01-25_23.png

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    "modmenu.summaryTranslation.iris": "Optifine의 셰이더\n§9✎§8 모드 토글, 셰이더 선택, 키바인드, ...",
+    "modmenu.summaryTranslation.itemmodelfix": "들고있는 아이템의 투명한 선 제거\n§9✎§8 메소드",
+    "modmenu.summaryTranslation.lambdabettergrass": "Optifine의 \"개선된 잔디/눈\"\n§9✎§8 퀄리티, 눈, 리소스 팩",
+    "modmenu.summaryTranslation.lambdynlights": "Optifine의 \"동적 조명\"\n§9✎§8 퀄리티, 엔티티 옵션, ...",
+    "modmenu.summaryTranslation.languagereload": "빠른 언어 교체, 검색 바\n§9✎§8 F3+J 로 언어 다시 로드",
+    "modmenu.summaryTranslation.lazydfu": "빠른 게임 시작\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.lithium": "물리와 청크 최적화\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.modmenu": "이 모드 메뉴\n§9✎§8 버튼 배치, 필터링, 정렬, ...",
+    "modmenu.summaryTranslation.morechathistory": "최대 채팅 히스토리 증가\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.nofade": "로딩 화면 Fade 애니메이션 제거\n§8✎ (옵션 없음)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/ko_kr.json:40]: Context: https://raw.githubusercontent.com/UltimateBoomer/mc-no-fade/assets/images/demo.gif

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    "modmenu.summaryTranslation.notenoughcrashes": "충돌 후에 리포트 생성/계속 플레이\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.phosphor": "§c*§r 조명, 청크 개선\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.puzzle": "일부 Optifine 기능\n§9✎§8 구성하려면 모드를 확대하세요 ( ▼ 아이콘 클릭)",
+    "modmenu.summaryTranslation.puzzle-base": "일부 Optifine 기능\n§9✎§8 로딩 화면 스플래시, ...",
+    "modmenu.summaryTranslation.smoothboot": "게임 응답 없음 감소\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.sodium": "렌더링, FPS 최적화\n§9✎§8 비디오 설정에서: 최적화, ...",
+    "modmenu.summaryTranslation.starlight": "조명, 청크 개선\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.reeses-sodium-options": "수직 비디오 설정\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.sodium-extra": "Optifine의 그래픽 설정\n§9✎§8 비디오 설정에서: 애니메이션, 파티클, ...",
+    "modmenu.summaryTranslation.tiefix": "Telementary 비활성화, 일부 오류 해결\n§9✎§8 해결할 오류, Telemantray 토글",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/ko_kr.json:50]:
```suggestion
    "modmenu.summaryTranslation.tiefix": "Telemetry 비활성화, 일부 오류 해결\n§9✎§8 해결할 오류, Telemetry 토글",
```

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    "modmenu.summaryTranslation.tooltipfix": "Tooltip overflow 차단\n§8✎ (옵션 없음)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/ko_kr.json:51]: See images on https://www.curseforge.com/minecraft/mc-mods/tooltipfix for context

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json (hunk 2 above)
Comment (AlphaKR93) [reply Mod Menu Helper/assets/modmenu/lang/ko_kr.json]: I couldn't remember it for a while during the translation, so I passed it over, but I forgot. I fixed it!

**Comment by AlphaKR93** [comment]
I changed all of them and improved some of the translations

File: Mod Menu Helper/assets/modmenu/lang/ko_kr.json [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    "modmenu.summaryTranslation.citresewn": "Optifine의 커스텀 아이템\n§9✎§8 모드 토글, 깨진 팩 허용, ...",
+    "modmenu.summaryTranslation.colormatic": "Optifine의 커스텀 색\n§9✎§8 맑은 하늘/공허, 부드러운 조명",
+    "modmenu.summaryTranslation.continuity": "Optifine의 연결된 텍스쳐\n§9✎§8 모드 토글, 메소드, 리소스 팩",
+    "modmenu.summaryTranslation.cullleaves": "Optifine의 \"스마트 나무\"\n§9✎§8 모드 토글, 리소스 팩",
+    "modmenu.summaryTranslation.cem": "Optifine의 커스텀 엔티티 (알파)\n§9✎§8 옵티파인 지원, 모델 고침",
+    "modmenu.summaryTranslation.dcch": "여러 월드에 걸쳐 입력된 기록 유지\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.dynamicfps": "백그라운드에서 FPS 감소\n§9✎§8 모드 토글, 호버시 복구, ...",
+    "modmenu.summaryTranslation.enhancedblockentities": "빠른 상자, 침대, 표지판 등 렌더\n§9✎§8 개선할 블록, 메소드",
+    "modmenu.summaryTranslation.entityculling": "벽 뒤 엔티티 렌더 안함\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.entity_texture_features": "Optifine의 랜덤/빛나는 몹\n§9✎§8 랜덤 몹, Emissivity, 눈 깜빡임, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/ko_kr.json:23]: "Emissivity" as in "whether it is emissive or not"

Comment (AlphaKR93) [reply Mod Menu Helper/assets/modmenu/lang/ko_kr.json]: fixed

**Comment by Madis0** [comment]
Looks good to me, thank you!

**Pull request merged**